examples/notebooks/healthcare/beam_nlp.ipynb

```diff
@@ -0,0 +1,968 @@
+        "         ],\n",
+        "    ),\n",
+        "    bigquery.SchemaField(\n",
+        "        \"certaintyAssessment\",\n",
+        "        \"RECORD\",\n",
+        "         mode=\"NULLABLE\",\n",
+        "         fields=[\n",
+        "             bigquery.SchemaField(\"value\", \"STRING\", mode=\"NULLABLE\"),\n",
+        "             bigquery.SchemaField(\"confidence\", \"FLOAT64\", mode=\"NULLABLE\"),\n",
+        "         ],\n",
+        "    ),\n",
+        "    bigquery.SchemaField(\n",
+        "        \"subject\",\n",
+        "        \"RECORD\",\n",
+        "         mode=\"NULLABLE\",\n",
+        "         fields=[\n",
+        "             bigquery.SchemaField(\"value\", \"STRING\", mode=\"NULLABLE\"),\n",
+        "             bigquery.SchemaField(\"confidence\", \"FLOAT64\", mode=\"NULLABLE\"),\n",
+        "         ],\n",
+        "    ),\n",
+        "    bigquery.SchemaField(\"confidence\", \"FLOAT64\", mode=\"NULLABLE\"),\n",
+        "    bigquery.SchemaField(\"id\", \"STRING\", mode=\"NULLABLE\")\n",
+        "]\n",
+        "\n",
+        "client = bigquery.Client()\n",
+        "\n",
+        "# Create Table IDs\n",
+        "table_ent = PROJECT+\".\"+DATASET+\".\"+TABLE_ENTITY\n",
+        "table_rel = PROJECT+\".\"+DATASET+\".\"+TABLE_REL\n",
+        "table_mentions = PROJECT+\".\"+DATASET+\".\"+TABLE_ENTITYMENTIONS\n",
+        "\n",
+        "# If table exists, delete the tables.\n",
+        "client.delete_table(table_ent, not_found_ok=True)\n",
+        "client.delete_table(table_rel, not_found_ok=True)\n",
+        "client.delete_table(table_mentions, not_found_ok=True)\n",
+        "\n",
+        "# Create tables\n",
+        "\n",
+        "table = bigquery.Table(table_ent, schema=schemaEntity)\n",
+        "table = client.create_table(table)  # Make an API request.\n",
+        "\n",
+        "print(\n",
+        "    \"Created table {}.{}.{}\".format(table.project, table.dataset_id, table.table_id)\n",
+        ")\n",
+        "\n",
+        "table = bigquery.Table(table_rel, schema=schemaRelations)\n",
+        "table = client.create_table(table)  # Make an API request.\n",
+        "print(\n",
+        "    \"Created table {}.{}.{}\".format(table.project, table.dataset_id, table.table_id)\n",
+        ")\n",
+        "table = bigquery.Table(table_mentions, schema=schemaEntityMentions)\n",
+        "table = client.create_table(table)  # Make an API request.\n",
+        "print(\n",
+        "    \"Created table {}.{}.{}\".format(table.project, table.dataset_id, table.table_id)\n",
+        ")"
+      ],
+      "metadata": {
+        "id": "bZDqtFVE5Wd_"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "**Pipeline Setup**\n",
+        "\n",
+        "For the purpose of experimenting, I have setup an interactiveRunner. But this should be changed to a DatafowRunner once you are comfortable with it"
```

Comment:
Why recommend Dataflow here? For notebooks the interactive runner seems like the best choice. 

also "DatafowRunner" -> "DataflowRunner"

Reply:
I agree, but I think if we want to standardize this we should write some documentation on notebooks guidelines. I can take this as a follow up AI.